const sub = (a, b) => {
    return new Promise((resolve, reject)=> {
        const subResult = a-b;
        if (subResult>=0) {
            resolve(subResult)
        } else {
            reject('wynik poniÅ¼ej 0')
        }
    })    
}


sub(3, 4)
    .then(result => {
        console.log(result)
    })
    .catch(error => {
        console.log(error)
    });

